cmd/rookflex/cmd/mount.go

```diff
@@ -134,6 +135,22 @@ func mountDevice(client *rpc.Client, mounter *k8smount.SafeFormatAndMount, devic
 			return fmt.Errorf("Rook: Mount volume failed. Error checking if %s is a mount point: %v", globalVolumeMountPath, err)
 		}
 	}
+
+	// Testing to see if the device is formatted. There has been observed an issue on slow systems where k8s believes there is no filesystem
+	// formatted even though one does exist. K8s then proceeds to format the volume which would result in data loss. This logging is an attempt
+	// to understand why k8s believes the volume is not formatted. See https://github.com/rook/rook/issues/1553
+	log(client, fmt.Sprintf("Testing to see if device %s needs formatting...", devicePath), false)
+	executor := &exec.CommandExecutor{}
+	output, err := executor.ExecuteCommandWithOutput(false, "", "blkid", "-p", "-s", "TYPE", "-s", "PTTYPE", "-o", "export", devicePath)
+	if err != nil {
+		log(client, fmt.Sprintf("Formatting test (blkid -p -s TYPE -s PTTYPE -o export %s). Device may not be formatted. err: %+v", devicePath, err), false)
```

Comment:
@jbw976 The error that prints when it isn't formatted includes the return code, so I think we're good to go. 
```
Failed to complete '': exit status 2.
```
While we could cast to get the error code, is it worth changing since we have the log message?

Reply:
oh okay, thank you for confirming that the exit code is included in the log output.  LGTM